packages/@aws-cdk/core/test/arn.test.ts

```diff
@@ -165,20 +173,61 @@ nodeunitShim({
           resource: 'identitypool',
           resourceName: 'us-east-1:1a1a1a1a-ffff-1111-9999-12345678:bla',
           sep: '/',
+          arnFormat: ArnFormat.SLASH_RESOURCE_NAME,
+          // cannot be successfully parsed with CFN functions because of the colons in resourceName
```

Comment:
In fact: not true! 

If we split on `/` first, then we can split on `:` on the LHS of that result!

But it may be a bit much to add to the current change. Let's just keep it like this and deal with the bug report when it comes.